### ISSUE TITLE
p100 power status default on

### ISSUE THREAD

**Issue by maxeq** [luisalvesmartins/TAPO-P100#1]
I have a problem, I really like Tapo  P100 and want to change my starting status.

My problem is that after a power outage, the p100 does not remember theprevious state and is always turned off.

How do I changeit ?  Is there any setting for a power outage?

**Comment by luisalvesmartins** [comment]
Not that I know of. The two workarounds I can think of are: 
1 - to be notified of a power outage and then turn it on automatically or 
2 - after turning it on having a deamon to periodically check if it's still on. 
Sorry for not being able to assist further.

**Comment by maxeq** [comment]
Yes, those workarounds are the same as what I was thinking about.

It is not possible to implement in code, right?
